src/main/resources/templates/springboot/rest/build.gradle.tmpl

```diff
@@ -32,12 +32,18 @@ dependencies {
 
     clientCompile "com.blackbaud:common-spring-boot-rest:\${springBootVersion}-2.+"
 
+    mainTestCompile "com.blackbaud:common-spring-boot-rest-test:\${springBootVersion}-2.+"
     mainTestCompile "com.blackbaud:common-test:1.+"
     mainTestCompile "org.codehaus.groovy:groovy-all:2.4.3"
     mainTestCompile ("org.spockframework:spock-spring:1.0-groovy-2.4") {
         exclude group: "org.codehaus.groovy"
     }
 
+    testCompile ('org.spockframework:spock-spring:1.0-groovy-2.4') {
```

Comment:
this is not necessary - testCompile inherits all dependencies from mainTestCompile